code/modules/admin/verbs/massmodvar.dm

```diff
@@ -28,10 +28,9 @@
 
 	var/list/locked = list("vars", "key", "ckey", "client")
 
-	for(var/p in forbidden_varedit_object_types)
-		if( istype(O,p) )
-			to_chat(usr, "<span class='danger'>It is forbidden to edit this object's variables.</span>")
-			return
+	if(!O.vv_edit_var(var_name))
+		to_chat(src, SPAN_WARNING("You cannot edit this variable."))
+		return
```

Comment:
I believe that this is used improperly as vv_edit_var is supposed to actually edit the variable and not (just) check if someone can edit the variable.

The usage like this might lead to the variable being edited ending up as NULL